src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -18,7 +18,10 @@
     @GeneratedValue(strategy= GenerationType.AUTO)
     private Long id;
 
+    @Column(name="status")
+    private String status;
+
     @ManyToOne
-    @JoinColumn(name="USER_ID")
+    @JoinColumn(name="userId")
```

Comment:
Było okej. kolumny nazywamy z dodaniem separatora _ , natomiast nazwy zmiennych z dużej litery (np. first_name i firstName)
https://www.xaprb.com/blog/2008/10/26/the-power-of-a-good-sql-naming-convention/

```suggestion
    @JoinColumn(name="user_id")
```